test/integration/smoke/test_ospf_zone_config.py

```diff
@@ -0,0 +1,165 @@
+
+        return
+
+    def setUp(self):
+        self.apiclient = self.testClient.getApiClient()
+        self.hypervisor = self.testClient.getHypervisorInfo()
+        self.dbclient = self.testClient.getDbConnection()
+        self.services = self.testClient.getParsedTestDataConfig()
+        self.zone = get_zone(self.apiclient, self.testClient.getZoneForTests())
+        self.pod = get_pod(self.apiclient, self.zone.id)
+        self.cleanup = []
+        return
+
+    def tearDown(self):
+        try:
+            # Clean up, terminate the created templates
+            cleanup_resources(self.apiclient, self.cleanup)
+        except Exception as e:
+            raise Exception("Warning: Exception during cleanup : %s" % e)
+        return
+
+    # Check vpcQuaggaConfigCmd positive test case
+    #VPCQuaggaConfigUpdateCmd [zoneid=1, quaggaProtocol=null, ospfArea=null, quaggaHelloInterval=null, quaggaDeadInterval=null, quaggaRetransmitInterval=null, quaggaTransitDelay=null, quaggaAuthentication=null, quaggaPassword=junk, ospfSuperCIDR=null, quaggaEnabled=null]
+    @attr(tags=["smoke", "advanced"], required_hardware="false")
+    def test_01_quaggaConfig(self):
+        
+        cmd = vpcQuaggaConfig.vpcQuaggaConfigCmd()
+        cmd.zoneid = self.zone.id
+        response = self.apiclient.vpcQuaggaConfig(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        
+        cmd = vpcQuaggaConfigUpdate.vpcQuaggaConfigUpdateCmd()
+        cmd.zoneid = self.zone.id
+        cmd.ospfSuperCIDR = "200.100.0.0/18,216.100.0.0/20"
+        cmd.quaggaEnabled = "true"
+        cmd.quaggaPassword = "6P33lsadEbmiJ7ZN7gK8vylOY2dTza3rAuY8UFboH7I=:Xb1G4VhzBkSXmTa8vDqExQQo8PiIUUSNtpxmdOkX1z8="
+        response = self.apiclient.vpcQuaggaConfigUpdate(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        
+        cmd = vpcQuaggaConfig.vpcQuaggaConfigCmd()
+        cmd.zoneid = self.zone.id
+        response = self.apiclient.vpcQuaggaConfig(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        cmd.zoneid = self.zone.id
+        
+        
+        self.assertEqual(
+                         response.quaggaEnabled, "true"
+                         )
+        
+        
+        self.assertEqual(
+                         response.quaggaPassword, "6P33lsadEbmiJ7ZN7gK8vylOY2dTza3rAuY8UFboH7I=:Xb1G4VhzBkSXmTa8vDqExQQo8PiIUUSNtpxmdOkX1z8="
```

Comment:
Consider extracting the password to a constant ensure consistency and increase clarity